kaggle_environments/envs/rps/rps.js

```diff
@@ -0,0 +1,110 @@
+async function renderer(context) {
+    const {
+        act,
+        agents,
+        environment,
+        frame,
+        height = 400,
+        interactive,
+        isInteractive,
+        parent,
+        step,
+        update,
+        width = 400,
+    } = context;
+
+    const weapon_names = ["Rock", "Paper", "Scissors", "Spock", "Lizard"]
```

Comment:
nit: `sign` instead of `weapon` here too

Reply:
Thanks, turns out I had "Find in *.py" checked so I missed all the non-py files 🤦